### PR TITLE
[DEV-4480] Uplift design on ask question page

### PR DESCRIPTION
**Pull request by empanda**
https://roverdotcom.atlassian.net/browse/DEV-4480

### CODE REVIEW

**Comment by empanda** [comment]
@mwhansen please review and +1. Thanks.

**Comment by mwhansen** [comment]
It looks like the styling is a bit off for administrators

screen shot 2015-07-28 at 9 21 01 am

**Comment by mwhansen** [comment]
Same with anonymous users

**Comment by mwhansen** [comment]
Actually, it seems better after force refreshing the CSS.

File: askbot/const/__init__.py [hunk 1]
```diff
@@ -9,8 +9,8 @@
 
 # Trying to access these from conf introduces an absolutely hideous
 # circular import.
-WORDS_TAG_SINGULAR = 'keyword'
-WORDS_TAG_PLURAL = 'keywords'
+WORDS_TAG_SINGULAR = 'tag'
```
Comment (mwhansen) [inline askbot/const/__init__.py]: Are we switching back from keywords to tags?

**Comment by mwhansen** [comment]
:+1: other than that tags / keywords question

File: askbot/const/__init__.py (hunk 1 above)
Comment (empanda) [inline askbot/const/__init__.py]: Yep, they want it to say `tag` now.

Comment (mwhansen) [inline askbot/const/__init__.py]: On the front page, there is still a "Keywords" section on the right bar.

Comment (empanda) [inline askbot/const/__init__.py]: I'll get that changed, thanks.

**Pull request merged**